<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sale_Widget</class>
 <widget class="QWidget" name="Sale_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>销售给管理系统</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>订单号</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>买家手机号</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>商品编号</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="Sale_lineEdit_order_id"/>
           </item>
           <item>
            <widget class="QLineEdit" name="Sale_lineEdit_buyer_tel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Sale_lineEdit_item_id"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>从</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="Sale_dateEdit_start">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>到</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="Sale_dateEdit_end">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="Sale_pushButton_select_order">
         <property name="text">
          <string>查询订单</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Sale_pushButton_select_number">
         <property name="text">
          <string>销售记录查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>订单(默认最近订单)</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTableView" name="tableView">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>销售记录(默认全商品)</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTableView" name="tableView_2">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="Sale_pushButton_new">
         <property name="text">
          <string>新建订单</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Sale_pushButton_change">
         <property name="text">
          <string>修改订单</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Sale_pushButton_delete">
         <property name="text">
          <string>删除订单</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Sale_pushButton_cancel">
         <property name="text">
          <string>取消订单</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Sale_pushButton_recive">
         <property name="text">
          <string>测试：确认签收</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Sale_pushButton_count">
         <property name="text">
          <string>销售记录盘点</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Sale_puushButton_revoke">
         <property name="text">
          <string>重置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Sale_pushButton_save">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
